File: forloop-add-test/forloop_add_test.go
package main

import (
    "os"
    "fmt"
    "strconv"
)

func main() {
    var i, j, toval, sumval int64
    toval, err := strconv.ParseInt(os.Args[1], 10, 64)
    if err != nil { return }
    sumval = 0
    for i=0; i<toval; i++ {
        for j=0; j<toval; j++ {
            sumval += i+(j^1)
        }
    }
    fmt.Println("Go:", sumval)
}

File: 1_forloop_add_test/src/forloop_add_test.java
class Forloop_add_test {
    public static void main(String[] args) {
        long toval = Long.parseLong(args[0], 10);
        long i, j, sumval = 0;
        for (i=0; i<toval; i++) {
            for (j=0; j<toval; j++) {
                sumval += i+(j^1);
            }
        }
        System.out.printf("Java: %d\n", sumval);
    }
}

